src/wallet/funding.rs

```diff
@@ -47,11 +47,13 @@ impl Wallet {
             fee_rate,
             manually_selected_outpoints,
         );
-        if ret.is_ok() {
-            log::info!(target: "wallet", "created funding txes random amounts");
-            return ret;
+
+        if ret.is_err() {
+            log::info!("Failed to create funding txes");
```

Comment:
this should be log::error. Also display the ret error value like before. Or else the log isn't very helpful.